src/components/TodoList.tsx

```
@@ -0,0 +1,43 @@
+        setOpen={setOpen}
+      ></UpdateTodo>
+
+      {todos.length === 0 ? (
+        <h1>No todos found..</h1>
+      ) : (
+        todos.map((todo) => (
+          <TodoItem
+            todo={todo}
+            setSelectedTodo={setSelectedTodo}
+            setOpen={setOpen}
+          ></TodoItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)